.github/workflows/ci.yaml

```diff
@@ -138,17 +124,6 @@ jobs:
       semver-latest: major
       run-tag: '1.4.4'
 
-# TODO: Need PR to add some gradle stuff in sdk-java to simplify this, but also we're not going
```

Comment:
While technically we could add this trigger from Java SDK to get master docker builds pushed to a repository I don't think master images are a part of our matrix.

I would want us to test against cloud on SDK main / master but that doesn't require building a docker image.

Reply:
(and shouldn't be defined as a part of this matrix so we're good)